<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Convert Tools</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="1">
    <widget class="QLineEdit" name="outputLed"/>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QComboBox" name="modelTypeCmb"/>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QComboBox" name="modelType2Cmb"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Model Output Destination</string>
     </property>
     <property name="buddy">
      <cstring>outputLed</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QComboBox" name="outputDTypeCmb"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Input Name</string>
     </property>
     <property name="buddy">
      <cstring>inputLed</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Output Data Type</string>
     </property>
     <property name="buddy">
      <cstring>outputDTypeCmb</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QToolButton" name="outputBtn">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QPushButton" name="convertBtn">
     <property name="text">
      <string>Convert</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="inputLed"/>
   </item>
   <item row="3" column="2">
    <widget class="QToolButton" name="inputBtn">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Model Type</string>
     </property>
     <property name="buddy">
      <cstring>modelTypeCmb</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Model Type</string>
     </property>
     <property name="buddy">
      <cstring>modelType2Cmb</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>modelTypeCmb</tabstop>
  <tabstop>modelType2Cmb</tabstop>
  <tabstop>inputLed</tabstop>
  <tabstop>inputBtn</tabstop>
  <tabstop>outputDTypeCmb</tabstop>
  <tabstop>outputLed</tabstop>
  <tabstop>outputBtn</tabstop>
  <tabstop>convertBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
